test/integration/test-amp-img.js

```diff
@@ -136,21 +136,64 @@ describe
     // IE doesn't support the srcset attribute, so if the developer
     // provides a srcset but no src to amp-img, it should set the src
     // attribute to the first entry in srcset.
-    it('should guarantee src if srcset is not supported', () => {
-      const imageLoadedPromise = waitForImageToLoad(fixture.doc);
-      return imageLoadedPromise.then(() => {
-        const ampImg = fixture.doc.getElementById('img4');
-        const img = ampImg.querySelector('img[amp-img-id="img4"]');
-        expect(img.getAttribute('src')).to.equal('/examples/img/hero@1x.jpg');
+    describe
+      .configure()
+      .ifIe()
+      .run('srcset support', () => {
+        it('should guarantee src if srcset is not supported', () => {
+          const imageLoadedPromise = waitForImageToLoad(
+            fixture.doc,
+            'img[amp-img-id="srcset"]'
+          );
+          return imageLoadedPromise.then(() => {
+            const ampImg = fixture.doc.getElementById('srcset');
+            const img = ampImg.querySelector('img[amp-img-id="srcset"]');
+            expect(img.getAttribute('src')).to.equal(
+              '/examples/img/hero@1x.jpg'
+            );
+          });
+        });
+      });
+
+    // IE can't scale SVG images, so intrinsic sizers use a PNG instead.
+    describe
+      .configure()
+      .enableIe()
+      .skipSafari()
```

Comment:
If we run this particular test on multiple browsers, it doesn't really make sense to be nested under "Rendering of amp-img - Internet Explorer edge cases".

Reply:
Renamed.